src/Destructurama.Attributed/Destructurama.Attributed.csproj

```diff
@@ -4,12 +4,23 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Use attributes to control how complex types are logged to Serilog.</Description>
     <RootNamespace>Destructurama</RootNamespace>
+    <IsPackable>true</IsPackable>
     <PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
+    <PackageId>Destructurama.Attributed</PackageId>
```

Comment:
```suggestion
```

The vast majority of the projects which I work on avoid this property.

Reply:
https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/389

Reply:
thanks